examples/basic-chat.html

```diff
@@ -0,0 +1,239 @@
+        color: #fff;
+        cursor: pointer;
+        border-radius: 0 8px 8px 0;
+      }
+      #send-btn:hover {
+        background-color: #43a047;
+      }
+
+      .messages {
+        max-height: 200px;
+        overflow-y: auto;
+        padding: 10px;
+        color: #fff;
+        white-space: pre-wrap;
+      }
+    </style>
+
+    <script>
+      // Called by Networked-Aframe when connected to server (optional)
+      // (this api will change in future versions)
+      function onConnect() {
+        console.log('onConnect', new Date());
+      }
+
+      // Note the way we're establishing the NAF schema here; this is a bit awkward
+      // because of a recent bug found in the original handling. This mitigates that bug for now,
+      // until a refactor in the future that should fix the issue more cleanly.
+      // see issue https://github.com/networked-aframe/networked-aframe/issues/267
+
+      // This one is necessary, because tracking the .head child component's material's color
+      // won't happen unless we tell NAF to keep it in sync, like here.
+      NAF.schemas.getComponentsOriginal = NAF.schemas.getComponents;
+      NAF.schemas.getComponents = (template) => {
+        if (!NAF.schemas.hasTemplate('#avatar-template')) {
+          NAF.schemas.add({
+            template: '#avatar-template',
+            components: [
+              // position and rotation are added by default if we don't include a template, but since
+              // we also want to sync the color, we need to specify a custom template; if we didn't
+              // include position and rotation in this custom template, they'd not be synced.
+              'position',
+              'rotation',
+
+              // this is how we sync a particular property of a particular component for a particular
+              // child element of template instances.
+              {
+                selector: '.head',
+                component: 'material',
+                property: 'color' // property is optional; if excluded, syncs everything in the component schema
+              }
+            ]
+          });
+        }
+        const components = NAF.schemas.getComponentsOriginal(template);
+        return components;
+      };
+
+      document.addEventListener('DOMContentLoaded', () => {
+        const form = document.querySelector('#form'); // form
+        const btn = document.querySelector('#send-btn'); // SEND btn
+        const input = document.querySelector('#input'); // input field with the text
+        const log = document.querySelector('.messages'); // message log
+        const username = document.querySelector('#user-name');
+        username.value = 'user-' + Math.round(Math.random() * 10000);
+
+        // when you want to send a message
+        form.addEventListener('submit', (evt) => {
+          evt.preventDefault();
+
+          // log your own messages
+          if (input.value === '') {
+            alert('Please enter a message');
```

Comment:
Instead of an alert, just do nothing. You could add `placeholder="Enter a message"` on the input, although it shows on Firefox it doesn't show on Chrome, really weird.